### PR TITLE
Don't uppercase the query string key

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: core/src/main/java/org/mapfish/print/map/image/wms/WmsUtilities.java [hunk 1]
```diff
@@ -62,6 +80,16 @@ public static URI makeWmsGetLayerRequest(
...
+        if (commonURI.getQuery() != null) {
+            StringTokenizer tokenizer = new StringTokenizer(commonURI.getQuery(), "&");
+            while (tokenizer.hasMoreTokens()) {
+                String token = tokenizer.nextToken();
+                String[] param = token.split("=");
+                if (param != null && param.length > 0 && param[0] != null) {
+                    extraParams.put(param[0], param[1]);
+                }
+            }
+        }
```
Comment (pvalsecc) [inline core/src/main/java/org/mapfish/print/map/image/wms/WmsUtilities.java]: You have a lib for that:
```java
for (NameValuePair pair: URLEncodedUtils.parse(commonURI, "UTF-8")) {
    extraParams.put(pair.getName(), pair.getValue());
}
```

Comment (sbrunner) [reply core/src/main/java/org/mapfish/print/map/image/wms/WmsUtilities.java]: Nice, thanks :-)

Comment (sbrunner) [reply core/src/main/java/org/mapfish/print/map/image/wms/WmsUtilities.java]: Done :-)

**Comment by pvalsecc** [comment]
/home/travis/build/mapfish/mapfish-print/core/src/main/java/org/mapfish/print/map/image/wms/WmsUtilities.java:25:8: Unused import - java.util.StringTokenizer.

**Comment by sbrunner** [comment]
:-P

**Comment by sbrunner** [comment]
Done :-)

**Pull request merged**